scene/2d/ray_cast_2d.cpp

```diff
@@ -224,6 +201,44 @@ void RayCast2D::_update_raycast_state() {
 		against = ObjectID();
 		against_shape = 0;
 	}
+
+	if (prev_collision_state != collided) {
+		_draw_debug_shape();
```

Comment:
Calling `_draw_debug_shape()` outside of NOTIFICATION_DRAW is causing errors, but it's enough to just call `update()` to force a redraw.

Reply:
Oops sorry, I forget to delete it.